rfc6962-bis.xml

```diff
@@ -1632,8 +1632,7 @@ certificates around the SCT timestamp.
     </section>
     <section title="Acknowledgements">
       <t>
-        The authors would like to thank Erwann Abelea, Robin Alden, Al Cutter,
-Francis Dupont, Adam Eijdenberg, Stephen Farrell, Daniel Kahn Gillmor, Brad Hill, Jeff Hodges, Paul Hoffman, Jeffrey Hutzelman, Stephen Kent, SM, Alexey Melnikov, Linus Nordberg, Chris Palmer, Trevor Perrin, Pierre Phaneuf, Melinda Shore, Ryan Sleevi, Carl Wallace and Paul Wouters for their valuable contributions.
+        The authors would like to thank Erwann Abelea, Robin Alden, Al Cutter, Francis Dupont, Adam Eijdenberg, Stephen Farrell, Daniel Kahn Gillmor, Brad Hill, Jeff Hodges, Paul Hoffman, Jeffrey Hutzelman, Stephen Kent, SM, Alexey Melnikov, Linus Nordberg, Chris Palmer, Trevor Perrin, Pierre Phaneuf, Melinda Shore, Ryan Sleevi, Carl Wallace, Paul Wouters, Kat Joyce, Martin Smith, Paul Hadfield and Pierre Phaneuf for their valuable contributions.
```

Comment:
Pierre was already in the list.  Also, please keep the names in alphabetical order.